src/current_user.ts

```diff
@@ -430,6 +434,28 @@ export default class CurrentUser {
     this.typingStateChange(eventPayload, roomId, onSuccess, onError);
   }
 
+  setCursor(
+    position: number,
+    room: Room,
+    onSuccess: () => void,
+    onError: (error: any) => void,
+  ) {
+    this.cursorsInstance
+      .request({
+        json: { position },
+        method: 'PUT',
+        path: `/cursors/0/rooms/${room.id}/users/${this.id}`,
```

Comment:
What does the `0` mean here?

Reply:
It's the "cursor type". Currently there's only one of them (read cursors). I did consider that it might be nicer to have more descriptive stringy ids... there's pros and cons to both. Do you have any thoughts either way?

Reply:
`0` feels a bit too magic number-y to me. What're the benefits we get from having a string-y version? Harder to slot it into an enum?

Reply:
Yeah, my initial thinking was that the cursor type (at this level at least) only needs to be machine readable, and having a string ID implies to me that new cursor types can be created dynamically, which they can't. How about defining the types as constants in the enum, and then slottinng them in to the url as `/cursors/${cursorTypes.read}/rooms/${room.id}/users/${this.id}`?

Reply:
Yeah that seems like a good solution to me 👍